core/src/main/java/com/tweet/core/model/Tweet.java

```diff
@@ -12,7 +9,10 @@
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private long tweetId;
-    private String username;
+
+    @ManyToOne
+    @JoinColumn(name = "username")
```

Comment:
lest modify this annotation a bit and add  @JoinColumn(name = "username", referencedColumnName = "username", nullable = false)